server/svix-server/src/lib.rs

```diff
@@ -135,3 +136,27 @@ pub async fn run(cfg: Configuration, listener: Option<TcpListener>) {
     worker_loop.expect("Error initializing worker");
     expired_message_cleaner_loop.expect("Error initializing expired message cleaner")
 }
+
+mod docs {
+    use axum::{
+        response::{Html, IntoResponse},
+        routing::get,
+        Json, Router,
+    };
+
+    pub fn router() -> Router {
+        Router::new()
+            .route("/docs", get(get_docs))
+            .route("/openapi.json", get(get_openapi_json))
+    }
+
+    async fn get_docs() -> Html<&'static str> {
+        Html(include_str!("static/docs.html"))
+    }
+
+    async fn get_openapi_json() -> impl IntoResponse {
```

Comment:
```suggestion
    async fn get_openapi_json() -> Result<Json> {
```

Reply:
Axum's Json requires a type signature :|

Reply:
OK, it doesn't matter that much, but you can just use `Result<Json<Json::Value>>` or whatever it is.